sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -38,6 +40,34 @@ export interface BearerTokenAuthenticationPolicyOptions {
     scopes: string | string[];
 }
 
+// @public
+export function bearerTokenChallengeAuthenticationPolicy(options: BearerTokenChallengeAuthenticationPolicyOptions): PipelinePolicy;
```

Comment:
move to `core-client`?  Would rest level clients use this?

Reply:
Yes in core-rest-pipeline.  Should behave the same as bearerTokenAuthenticationPolicy if no callbacks are provided.